spec/support/devise_token_auth_overrides.rb

```diff
@@ -2,11 +2,12 @@
 
 module DeviseTokenAuth
   module TokenFactory
```

Comment:
maybe this stuff can be removed now that we don't care about changes? or am I missing something?